src/Canaan.Kendryte.Flash.Shell/ViewModels/FlashViewModel.cs

```diff
@@ -253,7 +256,7 @@ public void BrowseFirmware()
             var dialog = new VistaOpenFileDialog
             {
                 Title = "Open firmware",
-                Filter = "Firmware (*.bin;*.kfpkg)|*.bin;*.kfpkg",
+                Filter = "Firmware (*.bin;*.kfpkg;*.zip)|*.bin;*.kfpkg;*.zip",
```

Comment:
Why do you add `.zip` here ?

Reply:
i'm lazy, .kfpkg need rename the packege.